src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogSink.cs

```diff
@@ -75,12 +78,13 @@ public DatadogSink(string apiKey, string source, string service, string host, st
 
             // Convert the JSON to a dictionnary and add the DataDog properties
             var logEventAsDict = JsonConvert.DeserializeObject<Dictionary<string, dynamic>>(payload.ToString());
-            logEventAsDict.Add("ddsource", source);
-            logEventAsDict.Add("service", service);
-            logEventAsDict.Add("host", host);
-            logEventAsDict.Add("ddtags", tags);
+            if ( source!= null) { logEventAsDict.Add("ddsource", source); }
```

Comment:
nits: `source!= null` -> `source != null` and same for all the following parameters